.github/workflows/readthedocs-builds.yml

```diff
@@ -0,0 +1,43 @@
+name: KFP Readthedocs Builds
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    paths:
+      - api/**
+      - sdk/**
+      - kubernetes_platform/**
+      - .github/workflows/readthedocs-builds.yml
+
+jobs:
+  test-readthedocs-builds:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
```

Comment:
(likely a bigger/out-of-scope issue) Python 3.9 hits EOS in less than 90 days, should we pre-empitively use 3.10+ here? Also, should we centralize this using `python-version-file` and a `.python-version`/`pyproject.toml` file?